news/2024/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks.md

```diff
@@ -0,0 +1,291 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+How did you learn about osu! and was your first impression of it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+I got introduced in an anime-related page I used to follow in Facebook! They were kinda just recommending it, and somehow that caught my interest, so I downloaded the game the same day (23 June, 2014). My first impression of the game was very positive, it was full of songs I already knew, plus the challenging aspect of the game. Just this was needed to **catch** me and make me want to improve in osu! and keep playing it for a long time!
+By the way, the image below is basically the exact post I found about osu!. I tried to find it again, but sadly I think the page is gone. Luckily I took a screenshot before, and saved it :D
+
+![](/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/osu_post_screenshot.jpg)
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Out of all game modes osu! has to offer, why did you choose osu!catch?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+I ended up getting closer to osu!catch than other modes because I started to have more contact with osu!catch players in the past than other game modes. Another reason might be because it felt more interesting to me to see myself (don't take it literal, it's the catcher, actually) "moving" in the screen and not clicking the notes in timing, or trying to aim (skill issue). It's a different feeling I have compared to other modes. I guess it feels like dancing to the beat XD. By the way, I don't dance IRL, I am SUPER shy...
```

Comment:
```suggestion
I ended up getting closer to osu!catch than other modes because I started to have more contact with osu!catch players in the past than other gamemodes. Another reason might be because it felt more interesting to me to see myself (don't take it literal, it's the catcher, actually) "moving" in the screen and not clicking the notes in timing, or trying to aim (skill issue). It's a different feeling I have compared to other modes. I guess it feels like dancing to the beat XD. By the way, I don't dance IRL, I am SUPER shy...
```